ApiDoc/OpenApiEndpointRouteDescriber.php

```diff
@@ -0,0 +1,197 @@
+<?php
+
+namespace Auto1\ServiceAPIHandlerBundle\ApiDoc;
+
+use Auto1\ServiceAPIComponentsBundle\Service\Endpoint\EndpointInterface;
+use Auto1\ServiceAPIComponentsBundle\Service\Endpoint\EndpointRegistryInterface;
+use Nelmio\ApiDocBundle\Describer\ModelRegistryAwareInterface;
+use Nelmio\ApiDocBundle\Describer\ModelRegistryAwareTrait;
+use Nelmio\ApiDocBundle\Model\Model;
+use Nelmio\ApiDocBundle\RouteDescriber\RouteDescriberInterface;
+use Nelmio\ApiDocBundle\RouteDescriber\RouteDescriberTrait;
+use OpenApi\Annotations\Get;
+use OpenApi\Annotations\Items;
+use OpenApi\Annotations\JsonContent;
+use OpenApi\Annotations\OpenApi;
+use OpenApi\Annotations\Operation;
+use OpenApi\Annotations\Parameter;
+use OpenApi\Annotations\PathItem;
+use OpenApi\Annotations\RequestBody;
+use OpenApi\Annotations\MediaType;
+use OpenApi\Annotations\Response;
+use OpenApi\Annotations\Schema;
+use OpenApi\Context;
+use OpenApi\Generator;
+use Symfony\Component\PropertyInfo\PropertyInfoExtractorInterface;
+use Symfony\Component\PropertyInfo\Type;
+use Symfony\Component\Routing\Route;
+
+/**
+ * Class EndpointRouteDescriber.
+ *
+ * @package Auto1\ServiceAPIHandlerBundle\ApiDoc
+ */
+class OpenApiEndpointRouteDescriber implements RouteDescriberInterface, ModelRegistryAwareInterface
```

Comment:
Do we need these changes in this ticket?

Reply:
yes, otherwise swagger file generation won't work for symfony 6+

Reply:
Does it mean that `EndpointRouteDescriber` will not work on Symfony 6?

Reply:
yes, that's why new one was written. Nelmio bundle 3 doesn't support symfony 6. Nelmio bundle v4 changed api, so for symfony <6 old `EndpointRouteDescriber` will be used and for symfony >=6 this new `OpenApiEndpointRouteDescriber` is used

Reply:
Will there be any documentation provided for end users?
I strongly believe that all BC-breaking changes should be documented

Reply:
I will add to Readme that swagger json file will be now `openapi: 3.0.0` instead of `swagger: 2.0`

Reply:
@aviator-ua I'm actually not sure we need to indicate such things.

Nelmio bundle is an external (not from this bundle) dependency. If a developer uses version 4+ they expect to have an output file in OpenAPI format. 

The only one important thing is that we support both formats and mix our data during the generation. So, nothing is broken